flow/src/main/resources/schema/flow/20221005170000_CB-18654_add_endtime_to_flowlog.sql

```diff
@@ -0,0 +1,8 @@
+-- // CB-18654 Add Flow Endtime to flowlog
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE IF EXISTS flowlog ADD COLUMN IF NOT EXISTS endtime int8;
```

Comment:
I have checked into the code on how this column is used and find the naming misleading. It is the last update timestamp on a flow state.